flink-ai-flow/examples/workflow_on_event/workflows/daily_workflow/daily_workflow.py

```diff
@@ -0,0 +1,101 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+import os
+from typing import List
+
+from ai_flow.workflow.control_edge import MeetAllEventCondition
+
+import ai_flow as af
+from ai_flow.api.context_extractor import ContextExtractor, EventContext, ContextList
+from ai_flow_plugins.job_plugins.python.python_processor import PythonProcessor, ExecutionContext
+from notification_service.base_notification import BaseEvent, ANY_CONDITION
+
+
+class DailyWorkflowContextExtractor(ContextExtractor):
+
+    def extract_context(self, event: BaseEvent) -> EventContext:
+        context_list = ContextList()
+        if event.event_type == 'DATA_EVENT' and 'hourly_processed_data' in event.key:
+            context_list.add_context(event.context)
+            return context_list
+
+
+class DataProcessingReader(PythonProcessor):
+
+    def process(self, execution_context: ExecutionContext, input_list: List) -> List:
+        day = execution_context.job_execution_info.workflow_execution.context
+        dataset_meta: af.DatasetMeta = execution_context.config.get('dataset')
+        daily_dir = os.path.join(dataset_meta.uri, day)
+        file_names = os.listdir(daily_dir)
+        return [[os.path.join(daily_dir, file_name) for file_name in file_names]]
+
+
+class DataProcessingProcessor(PythonProcessor):
+
+    def process(self, execution_context: ExecutionContext, input_list: List):
+        file_paths = input_list[0]
+        res = 0
+        for file_path in file_paths:
+            with open(file_path, 'rt') as f:
```

Comment:
't' is a redundant config for open() method as text model is the default choice.

Reply:
fixed in 7241764